commit 74e2413f18ba929dcd21b35c6886a391c05358b9
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Fri Mar 10 02:39:25 2017 +0100

    Resolve file collisions with KF5Akonadi{,Mime}, KF5MailTransport
    
    mv resourcebase.kcfg -> pim4resourcebase.kcfg
    mv specialmailcollections.kcfg -> pim4specialmailcollections.kcfg
    mv mailtransport.kcfg -> pim4mailtransport.kcfg

diff --git a/akonadi/CMakeLists.txt b/akonadi/CMakeLists.txt
index 1d32b3f92..3bd0c8290 100644
--- a/akonadi/CMakeLists.txt
+++ b/akonadi/CMakeLists.txt
@@ -261,7 +261,7 @@ qt4_add_dbus_adaptor( akonadikde_LIB_SRC interfaces/org.freedesktop.Akonadi.Agen
 
 
 kde4_add_kcfg_files( akonadikde_LIB_SRC resourcebasesettings.kcfgc )
-install( FILES resourcebase.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
+install( FILES pim4resourcebase.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
 
 kde4_add_ui_files( akonadikde_LIB_SRC
   cachepolicypage.ui
diff --git a/akonadi/kmime/CMakeLists.txt b/akonadi/kmime/CMakeLists.txt
index 23bace385..bd2b9fe6c 100644
--- a/akonadi/kmime/CMakeLists.txt
+++ b/akonadi/kmime/CMakeLists.txt
@@ -79,7 +79,7 @@ set( kmimeakonadi_LIB_SRC
 endif()
 
 kde4_add_kcfg_files( kmimeakonadi_LIB_SRC specialmailcollectionssettings.kcfgc )
-install( FILES specialmailcollections.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
+install( FILES pim4specialmailcollections.kcfg DESTINATION ${KCFG_INSTALL_DIR} )
 
 include_directories(
   ${CMAKE_BINARY_DIR}/kmime
diff --git a/akonadi/kmime/specialmailcollections.kcfg b/akonadi/kmime/pim4specialmailcollections.kcfg
similarity index 100%
rename from akonadi/kmime/specialmailcollections.kcfg
rename to akonadi/kmime/pim4specialmailcollections.kcfg
diff --git a/akonadi/kmime/specialmailcollectionssettings.kcfgc b/akonadi/kmime/specialmailcollectionssettings.kcfgc
index 429ee12e9..e9e1a55c0 100644
--- a/akonadi/kmime/specialmailcollectionssettings.kcfgc
+++ b/akonadi/kmime/specialmailcollectionssettings.kcfgc
@@ -1,4 +1,4 @@
-File=specialmailcollections.kcfg
+File=pim4specialmailcollections.kcfg
 ClassName=SpecialMailCollectionsSettings
 NameSpace=Akonadi
 Singleton=true
diff --git a/akonadi/resourcebase.kcfg b/akonadi/pim4resourcebase.kcfg
similarity index 100%
rename from akonadi/resourcebase.kcfg
rename to akonadi/pim4resourcebase.kcfg
diff --git a/akonadi/resourcebasesettings.kcfgc b/akonadi/resourcebasesettings.kcfgc
index 685637b55..df025beac 100644
--- a/akonadi/resourcebasesettings.kcfgc
+++ b/akonadi/resourcebasesettings.kcfgc
@@ -1,4 +1,4 @@
-File=resourcebase.kcfg
+File=pim4resourcebase.kcfg
 ClassName=ResourceBaseSettings
 NameSpace=Akonadi
 Singleton=true
diff --git a/mailtransport/CMakeLists.txt b/mailtransport/CMakeLists.txt
index c86514d63..15c351eca 100644
--- a/mailtransport/CMakeLists.txt
+++ b/mailtransport/CMakeLists.txt
@@ -102,7 +102,7 @@ set_target_properties(mailtransport PROPERTIES
 )
 
 install(TARGETS mailtransport EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
-install(FILES mailtransport.kcfg DESTINATION ${KCFG_INSTALL_DIR})
+install(FILES pim4mailtransport.kcfg DESTINATION ${KCFG_INSTALL_DIR})
 
 set(mailtransport_kcm_srcs configmodule.cpp)
 kde4_add_plugin(kcm_mailtransport ${mailtransport_kcm_srcs})
diff --git a/mailtransport/mailtransport.kcfg b/mailtransport/pim4mailtransport.kcfg
similarity index 100%
rename from mailtransport/mailtransport.kcfg
rename to mailtransport/pim4mailtransport.kcfg
diff --git a/mailtransport/transportbase.kcfgc b/mailtransport/transportbase.kcfgc
index 950cb7516..3116bd5a0 100644
--- a/mailtransport/transportbase.kcfgc
+++ b/mailtransport/transportbase.kcfgc
@@ -1,4 +1,4 @@
-File=mailtransport.kcfg
+File=pim4mailtransport.kcfg
 ClassName=TransportBase
 NameSpace=MailTransport
 Mutators=true
